api/client/build.go

```diff
@@ -84,8 +86,37 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		context  io.ReadCloser
 		isRemote bool
 		err      error
+		outBuf   io.Writer
 	)
 
+	// outBuf is for the Client's verbose output
+	// If script mode is set, then this output should be ignored
+	if !*scriptMode {
+		outBuf = cli.out
+	} else {
+		// First check whether the Docker daemon supports this feature.
+		serverResp, err := cli.call("GET", "/version", nil, nil)
```

Comment:
@duglin, no problem. I will remove it. Although, I will propose this check later.